frontend/sozisel-app/src/components/PresenterSession/EventsTimline/LiveEventDetails/WhiteboardLiveEventDetails/WhiteboardLiveEventDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement, useContext } from "react";
+import { LiveEventContext } from "../LiveEventDetails";
+import TextSection from "../TextSection";
+import Timer from "../Timer";
+import useFetchEventLiveResult from "../../../../../hooks/useFetchEventLiveResult/useFetchEventLiveResult";
+import { useTranslation } from "react-i18next";
+
+const WhiteboardLiveEventDetails = (): ReactElement => {
+  const { t } = useTranslation("common");
+  const { activeEvent, event, sessionId, onFinishCallback } =
+    useContext(LiveEventContext);
+
+  const { finishedTrials } = useFetchEventLiveResult(
+    sessionId,
+    event,
+    activeEvent.id,
+    "WhiteboardResult"
+  );
+
+  if (event.eventData.__typename !== "Whiteboard") {
```

Comment:
can we have some helper function that extracts those `eventData.__typename` and transforms it to some enum and replace it everywhere we use that?